test/TritonGPU/amd/accelerate-amd-matmul-wmma.mlir

```diff
@@ -49,4 +49,30 @@ module attributes {"triton_gpu.num-ctas" = 1 : i32, "triton_gpu.num-warps" = 4 :
     tt.store %2, %4 : tensor<32x32x!tt.ptr<f16>, #blocked>
     tt.return
   }
+  tt.func public @wmma_dot_ab8_cf16(
+   // CHECK: %[[DOT2_ARG_A:.+]]: tensor<32x128xf8E5M2, #triton_gpu.dot_op<{opIdx = 0, parent = #[[DOT_OP_PARENT]]}>>
```

Comment:
For LIT test we want to be checking the bits that are relevant. Here I think we don't need to specify the full line for the args as they won't change in any sense. Just `%[[DOT2_ARG_A:.+]]: tensor<32x128xf8E5M2{{.+}}>` should be enough

Reply:
I'm doing it to exact identify the variable to use it further.
I think, removing some info from this check will make test more unstable and sensitive to changes.
for example, ``[[DOT2_ARG_A:.+]]: tensor<32x128xf8E5M2{{.+}}>`` could be also matched with tt.dot from result and also with value after conversion.

Reply:
Okay. It's a small issue anyway.